core/src/main/java/feign/Logger.java

```diff
@@ -107,7 +107,7 @@ protected Response logAndRebufferResponse(String configKey, Level logLevel, Resp
     return response;
   }
 
-  protected IOException logIOException(String configKey, Level logLevel, IOException ioe, long elapsedTime) {
+  protected void logIOException(String configKey, Level logLevel, IOException ioe, long elapsedTime) {
```

Comment:
not good enough reasons, as this would break people who are extending this regardless.

a good enough reason is that this method was exposed protected, but not yet released! (unless I'm mistaken) This means that no one would have been able to extend it yet, so we wouldn't be breaking anyone.